### DISCUSSION TITLE
mantine select with nextjs formik

### DISCUSSION THREAD

**Discussion by Lalman888** [mantinedev/mantine#4521] [Q&A]
I want to give select input name and id but its not working 

<Select
      placeholder={placeholder}
      searchable
      nothingFound="No options"
      data={data}
      rightSection={<ChevronDown size={18} />}
      onChange={onChange}
      value={value}
      name={name}
      id={name}
    />

**Comment by ItaiAxelrad** [comment]
You'll need a binding like formik-mantine [https://codesandbox.io/s/formik-mantine-showroom-46h1jx?file=/src/App.tsx](https://codesandbox.io/s/formik-mantine-showroom-46h1jx?file=/src/App.tsx). Might I suggest you use the [forms package](https://mantine.dev/form/use-form/) from Mantine instead if your needs are simple

**Comment by Lalman888** [comment]
In mantine I get the issue with input to change the UI I need to use !important in CSS

**Comment by ItaiAxelrad** [comment]
What does styling have to do with formik? Do you have a code example of the issue?
